draft-ietf-rtcweb-jsep.xml

```diff
@@ -2293,6 +2284,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             section is bundled into another m= section, it still MUST
             NOT contain any ICE credentials.</t>
 
+            <t>If the m= section is not bundled into another
```

Comment:
Sorry for being unclear - I meant for the discussion of the post-answer state to go into the paragraph below that starts with "
   If the initial offer was applied using setLocalDescription, and an
   answer from the remote side has been applied using
   setRemoteDescription, meaning the PeerConnection is in the "remote-
   pranswer" or "stable" states"